src/components/applications/DevApplicationDetails.js

```diff
@@ -0,0 +1,106 @@
+                    <FileViewerWrapper undecodedFile={file}/>
+                </Grid>))
+    }
+
+    useEffect(() => {
+        tasksApi.getTasksFromOrganization(devApplication.organizationUUID, devPassword)
+            .then(data => setOrganizationTasks(data))
+        tasksApi.getTasksFromTaskStage(devApplication.taskStageUUID, devPassword)
+            .then(data => setAlreadyAssignedTasks(data))
+    }, [devApplication, reload])
+
+    const assignTask = () => {
+        setOpenAssignTasks(true)
+    }
+
+    const getCardContent = () => {
+        return (<CardContent>
+            <Grid container direction="row" spacing={4}>
+                <Grid item xs={12}>
+                    <Grid container spacing={2}>
+                        <Grid item xs={12}
+                              style={{display: "flex", flexDirection: "row", justifyContent: "space-between"}}>
+                            <div>
+                                <Typography
+                                    variant="h6">{application.jobSeeker.user.firstName} {application.jobSeeker.user.lastName}</Typography>
+                            </div>
+                            <div>
+                                <Typography variant="h6" color="textSecondary">{application.offerName || devApplication?.position}</Typography>
+                            </div>
+                        </Grid>
+                        <Grid item xs={12}
+                              style={{display: "flex", flexDirection: "row", justifyContent: "space-between"}}>
+                            <div>
+                                <Typography>{application.jobSeeker.user.mail}</Typography>
+                            </div>
+                            <div>
+                                <Typography color="textSecondary">{processAPI._prepareDate(new Date(application.applicationDate))}</Typography>
+                            </div>
+                        </Grid>
+                        <Grid item xs={12}><Divider/></Grid>
+                        <Grid item xs={8}>
+                            {
+                                application.seekerFiles.length === 0 ?
+                                    <Typography>Candidate didn't supply any files.</Typography> :
+                                    <Grid container direction="row" spacing={4}>
+                                        {getSeekerFiles()}
+                                    </Grid>
+                            }
+                        </Grid>
+                        <Grid item xs={12}>
+{/*                         UNCOMMENT AFTER MERGE WITH NOTES - ES-231 I THINK SO
+                            <div style={{display: 'flex', gap: '1em', flexFlow: 'row wrap'}}>
+                            {
+                                notes.map(note => <SingleNote note={note} />)
+                            }
+                            </div>
+                             */}
```

Comment:
Ten fragment w takim razie nie będzie rebaseowany jak tamto zostanie zmergowane?

Reply:
Gdybym jeszcze ogarniał rebase'a, to może bym wiedział, jak to zrobić xD

Reply:
Znaczy merge od rebase nie różni się jakoś super, ale nie ma znaczenia. Pytanie było czy ten fragment będzie w ES-231?